logging/internal/common.go

```diff
@@ -16,14 +16,20 @@ package internal
 
 import (
 	"fmt"
+	"runtime"
 	"strings"
+	"sync"
+	"unicode"
 )
 
 const (
 	// ProdAddr is the production address.
 	ProdAddr = "logging.googleapis.com:443"
 )
 
+// InstrumentOnce guards instrumenting logs one time
+var InstrumentOnce = new(sync.Once)
```

Comment:
Why is this declared in internal? Feels like it should live next to the code it intends to gaurd and/or be wrapped in a struct. Either this should move or the code that is being guarded should move here.

This can just be `var InstrumentOnce sync.Once`